include/cuco/static_multimap.cuh

```diff
@@ -1342,9 +1353,11 @@ class static_multimap {
   }
 
  private:
-  std::size_t capacity_{};                      ///< Total number of slots
-  Key empty_key_sentinel_{};                    ///< Key value that represents an empty slot
-  Value empty_value_sentinel_{};                ///< Initial value of empty slot
+  std::size_t capacity_{};        ///< Total number of slots
+  Key empty_key_sentinel_{};      ///< Key value that represents an empty slot
+  Value empty_value_sentinel_{};  ///< Initial value of empty slot
+  // TODO multimap erase
```

Comment:
So for now the sentinel has been introduced to multimap because the size computation uses the common kernel?

Reply:
Right, we would eventually add this feature thus putting a placeholder as a reminder (also for the convenience of common kernel)